lib/chat_api_web/router.ex

```diff
@@ -114,6 +114,7 @@ defmodule ChatApiWeb.Router do
     resources("/tags", TagController, except: [:new, :edit])
     resources("/browser_sessions", BrowserSessionController, except: [:create, :new, :edit])
     resources("/personal_api_keys", PersonalApiKeyController, except: [:new, :edit, :update])
+    resources("/canned_responses", CannedResponseController)
```

Comment:
nit - we should exclude `:new` and `:edit` here:
```elixir
resources("/canned_responses", CannedResponseController, except: [:new, :edit])
```

("new" and "edit" are primarily used for full-stack phoenix applications, where we render those pages on the server... for a JSON API they aren't necessary :+1:)